### PR TITLE
new code

### PR DESCRIPTION
**Pull request by AnnaZhao0209**
just push new code

### CODE REVIEW

**Comment by EvaGao88** [comment]
:)

**Comment by AnnaZhao0209** [comment]
> :)

thank you

**Review by qingyuanre1 — APPROVED** [review]
Good job !

File: src/test/java/GitPushTest.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	public static void main(String[] args) {
+		System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver");
+		 driver = new ChromeDriver();
+		try {
+			driver.navigate().to("https://jqueryui.com/");
+			driver.findElement(By.linkText("Menu")).click();
+			driver.switchTo().frame(0);
+			driver.findElement(By.id("ui-id-2")).click();
+			
+			System.out.println("Pass haha, I would like to see pass :)");
```
Comment (AnnaZhao0209) [inline src/test/java/GitPushTest.java:16]: I will cancel line 16

**Comment by shuqin32** [comment]
good job

File: src/test/java/GitPushTest.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		} catch (Exception e) {
+			e.printStackTrace();
+		}finally {
+			driver.quit();
+		}
+		
+		
+		
+
+}}
```
Comment (minyizeng) [inline src/test/java/GitPushTest.java:27]: you should delete one " } "  on line 27

**Review by CarterCChen — APPROVED** [review]
looks good

**Comment by shuqin32** [comment]
good job

**Pull request closed without merging**